### PR TITLE
[Feature] Add survey link

### LINKED ISSUE #9551: ♻️ Update application confirmation copy and add link to survey
image

## ✨ Summary

[Visit the Figma file](https://www.figma.com/file/i7FhmyCoacRFSPmK6b7h7C/Application-confirmation?type=design&node-id=2%3A3&mode=design&t=Cmfg0WlS0PLVKyQT-1) • Contact @substrae with design questions

This ticket aims to update a few small copy elements on the application confirmation page as well as add a new link to our post-application survey.

```[tasklist]
### ✅ Requirements
- [ ] Update the next step link labels to match
- [ ] Add the new "Tell us about your experience" button and link it to the survey ([English](https://forms-formulaires.alpha.canada.ca/en/id/clsdw2j7t00igxd8211b53rlv), [French](https://forms-formulaires.alpha.canada.ca/fr/id/clsdw2j7t00igxd8211b53rlv))
- [ ] Ensure the survey link opens in a new tab
- [ ] Change the existing dashboard button label and swap it to an 'inline' style button
- [ ] Triple check dark mode
```

```[tasklist]
### 🔗 Related
- [ ] #9550 
``` 

## 🔍 Details

<details>
  <summary>📸 Full page (light)</summary>
  
</details> 

<details>
  <summary>📸 Full page (dark)</summary>
  
</details>

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #9551 

## 👋 Introduction

Adds the survey link to the application success page as well as adjusting some copy for links.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Build `npm run dev`
2. Login as applicant `applicant@test.com`
3. Complete an application
4. Confirm the links in the list have the copy updated
5. Confirm the new survey link exists and link to the proper English and French pages

## 📸 Screenshot

Screenshot 2024-03-04 150820
Screenshot 2024-03-04 150828

### CODE REVIEW

File: apps/e2e/cypress/e2e/talentsearch/submit-application-iap.cy.ts [hunk 1]
```diff
@@ -310,7 +310,7 @@ describe("Submit Application for IAP Workflow Tests", () => {
       "Thank you for your interest in becoming an IT apprentice with the Government of Canada.",
     ); // customized copy for IAP
     cy.findByRole("link", {
-      name: /Visit your Profile and applications page/i,
+      name: /return yo your dashboard/i,
```
Comment (mnigh) [inline apps/e2e/cypress/e2e/talentsearch/submit-application-iap.cy.ts:313]:
```suggestion
      name: /return to your dashboard/i,
```

File: apps/web/src/pages/Applications/ApplicationSuccessPage/ApplicationSuccessPage.tsx [hunk 2]
```diff
@@ -139,39 +138,61 @@ const ApplicationSuccess = ({ application }: ApplicationPageProps) => {
               data-h2-vertical-align="base(top)"
             >
               {intl.formatMessage({
-                defaultMessage:
-                  "Submit an application to ongoing recruitment talent pools.",
-                id: "ZTnze/",
+                defaultMessage: "Browse for other opportunities",
+                id: "nc5GGm",
```
Comment (vd1992) [inline apps/web/src/pages/Applications/ApplicationSuccessPage/ApplicationSuccessPage.tsx:142]: Does it still make sense then to link to `/en/browse/pools#ongoingRecruitments` or should it just point to the top of the browse page?

Comment (esizer) [reply apps/web/src/pages/Applications/ApplicationSuccessPage/ApplicationSuccessPage.tsx:142]: That is a good question. I'm not sure how to answer it though 😅 @substrae any thoughts?

Comment (substrae) [reply apps/web/src/pages/Applications/ApplicationSuccessPage/ApplicationSuccessPage.tsx:142]: @vd1992 @esizer 

Yep good catch - just link to `/browse`

Comment (esizer) [reply apps/web/src/pages/Applications/ApplicationSuccessPage/ApplicationSuccessPage.tsx:142]: Thanks! Updated in [remove hash from browse link](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9632/commits/7490534a2fe3b9e61c3be5de954cdc5dd3e9be93)

**Review by vd1992 — APPROVED** [review]
Looking good!

**Pull request merged**